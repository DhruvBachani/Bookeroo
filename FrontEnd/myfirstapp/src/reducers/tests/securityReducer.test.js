import React from 'react';
import reducer from '../../../src/reducers'
import { shallow } from 'enzyme';
import { initialState } from '../securityReducer';

describe('securityReducer', () => {
    it('should return the initial state', () => {
        expect(reducer(undefined, {})).toMatchSnapshot();
    })

    // it('should handle SET_CURRENT_USER', () => {
    //     expect(
    //         reducer(initialState,
    //             {
    //                 type: 'SET_CURRENT_USER'
    //             })
    //     ).toMatchSnapshot();
    // })
})