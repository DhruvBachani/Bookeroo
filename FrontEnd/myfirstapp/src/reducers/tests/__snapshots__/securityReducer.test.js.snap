// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`securityReducer should return the initial state 1`] = `
Object {
  "book": Object {
    "allCategories": Array [],
    "book": Array [],
    "books": Array [],
  },
  "errors": Object {},
  "person": Object {
    "person": Object {},
    "persons": Array [],
  },
  "requests": Object {
    "userRequests": Array [],
  },
  "review": Object {
    "bookReviews": Array [],
  },
  "security": Object {
    "user": Object {},
    "validToken": false,
  },
}
`;
